packages/flutter_tools/test/web.shard/hot_reload_web_test.dart

```diff
@@ -48,7 +48,7 @@ Future<void> _testProject(HotReloadProject project, {String name = 'Default'}) a
 
   testWithoutContext('$testName: hot restart works without error', () async {
     flutter.stdout.listen(printOnFailure);
-    await flutter.run(chrome: true, additionalCommandArgs: <String>['--verbose', '--web-renderer=html']);
+    await flutter.run(chrome: true, additionalCommandArgs: <String>['--verbose']);
```

Comment:
I'm going to restore the `html` renderer in these tests, to see if they cure the broken test, but that's definitely not a permanent fix :/